### ISSUE TITLE
option to remove system variables and score/calculated fields from the data

### ISSUE THREAD

**Issue by aghaynes** [SwissClinicalTrialOrganisation/secuTrialR#252]
**Is your feature request related to a problem? Please describe.**

Most system variables are not useful and just take up space... 

calculated fields should probably not be used in stats and should be recalculated instead

Both of these field types could thus be removed 

**Describe the solution you'd like**
option to read_secutrial to drop them?

**Comment by DrEspresso** [comment]
Good idea, but there are a few system variables I find useful for joining eCRFs and repetition tables, or identifying latest AE followups (`mnpvislabel`, `mnpvisno`, `mnpdocid`, `mnpsubdocid`, `mnpaeid`, `mnpaefuid`). The option could thus be multi-sided, e.g. "none", "all", "minimal", if other users also find these or similar variables useful.

**Comment by aghaynes** [comment]
indeed, it's certainly not appropriate to remove everything under all circumstances... your suggestion might well be one way around it
